package.json

```diff
@@ -47,7 +49,9 @@
     "build:storybook": "storybook build -o build/storybook",
     "storybook": "storybook dev -p 6006",
     "test": "react-scripts test",
-    "lint": "eslint src/",
+    "lint:prettier": "prettier . --check",
+    "lint": "npm run lint:prettier",
```

Comment:
Ooh wait, I had missed this! We don't want `lint` to just run Prettier, we want it to run ESLint still at a minimum. Do we _also_ want it to run Prettier? I… actually don't know. cc @slifty what do you think?

The path forward is probably either:

1. Change `lint` back to `eslint src/` (and… add `lint:prettier` to CI?)
2. Change `lint` to `eslint src/ && npm run lint:prettier`?

Reply:
Update: Ooh, I see [what `service` did](https://github.com/PhilanthropyDataCommons/service/pull/695/commits/f6c7b400920657e2198b138f22103bbb5c578782#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519), and I like it. Let's copy that! So we actually need to add a `lint:ts` action that runs ESLint, then have our `lint` action run both `lint:ts` and `lint:prettier`.

For front-end, I think we probably need to lint both `.ts` and `.tsx` files… and it may not hurt to include `.js` files? Again @slifty what do you think. (The only `.js` files we have were generated by our OIDC library and live in `public/`.)

Reply:
yes! where you landed!